doc/semantics-of-the-publishing-api.md

```diff
@@ -0,0 +1,406 @@
+## Publishing API V2 Semantics
+
+This document explains the semantics of Publishing API V2. This includes:
+
+- What do each of the fields mean
+- What values are allowed in each field
+- What is the effect of choosing one value over another
+
+There is another document that explains the syntax for how the API should be
+used. That document can be found
+[here](https://docs.google.com/document/d/1zYFZFZ2TLzasgX4SZTxuQhLzqRYCk4hWKBTcb8QFIRA/edit).
+
+---
+
+### **Required Fields**
+
+---
+
+### content_id
+
+Example: *d296ea8e-31ad-4e0b-9deb-026da695bb65*
+
+Required: Yes
+
+The `content_id` is the content item’s main identifier and it forms its identity
+as it travels through the pipeline. This is why requests to create and query
+content items tend to be keyed by `content_id` in the URL of the request.
+
+Each `content_id` refers to a single piece of content, with a couple of caveats:
+
+- `content_ids` are shared across locales – the English and French versions of a
+content item share a `content_id`
+
+- `content_ids` are shared across publish states – the draft and live versions of
+a content item share a `content_id`
+
+`content_ids` are [UUIDs](https://en.wikipedia.org/wiki/Universally_unique_identifier)
+and will not be accepted by the Publishing API otherwise.
```

Comment:
Do we need to specify a format for the content ID?  Do we validate it anywhere other than local tests?